src/main/resources/Documentation/about.md

```diff
@@ -1,41 +1,55 @@
-TODO: Add Plugin introduction
+Eiffel Gerrit plugin is may be used to send Eiffel events when changes occurs in in a project in Gerrit.
+This plugin simplifies communication in continuous integration and delivery pipelines and helps keeping
+source code management traceability.
 
 For more Eiffel information follow this link:
 <https://github.com/eiffel-community/eiffel-gerrit-plugin>
 
+#####Eiffel Events:
+
+Version 0.0.1
+:   SourceChangeCreatedEvent
+    SourceChangeSubmittedEvent
+
 Parameters
 ----------
 
 Project Configuration:
 
 eiffel-integration.enabled
-:   Plugin state. if set to `true`, send message Eiffel message for every commit submitted for this repository.
-    Plugin considered disabled if this option is not defined or set to `false`.
+:   If set to `true` the plugin will send eiffel events.
+    Is set to `false` or is not defined no eiffel events will be sent.
 
 eiffel-integration.filter
-:   Allow users to defined a white list of branches, i.e. messages will be sent only for a commits submitted on specified branches, all other submits will be ignored.
-    Accepts Java regex's separated by spaces. If not defined, send messages for commits submitted on all branches.
+:   Allow users to defined a white list of branches, i.e. messages will be sent only for source change on
+    specified branches. Source Change to any other branch will be ignored. Accepts full branch name and or
+    regex separated by spaces. If not defined, messages will be sent for source changes in all branches.
+    See Example configuration where release branch may be triggered on release-(Any version).
 
 eiffel-integration.flow-context
 :   Optional, but expects a UUID of a EiffelFlowContextDefinedEvent if provided.
-    Can accept List of UUID's seperated by `,` that tell which flow it will be part of. (More about Flow Context: <https://github.com/eiffel-community/eiffel/blob/master/eiffel-vocabulary/EiffelSourceChangeCreatedEvent.md#flow_context>) 
+    Can accept List of UUID's seperated by `,` that tell which flow it will be part of. (More about Flow Context:
```

Comment:
seperated  --> separated 
tell --> tells
"... which flow it will be part of."  it in this case is the Eiffel event right? Maybe clarify that.

Reply:
Done in next commit.